Programmers/230919_롤케이크_자르기/kwaksj329.js

```diff
@@ -0,0 +1,25 @@
+function solution(topping) {
+  let answer = 0;
+  let toppingSize = topping.length;
+  let brother = new Map();
+  const sister = new Set();
+  topping.forEach((value) => {
+    brother.has(value)
+      ? brother.set(value, brother.get(value) + 1)
+      : brother.set(value, 1);
```

Comment:
```suggestion
    brother.set(value, (brother.get(value) ?? 0) + 1);
```
이렇게 줄여볼 수 있을 것 같아요 ㅎㅎ